Makefile

```diff
@@ -302,6 +305,14 @@ run-proxy:
 run-build-vm: $(BUILD_VM) $(BIOS_IMG) $(DEVICETREE_DTB)
 	$(QEMU_SYSTEM) $(QEMU_OPTS) -drive format=qcow2,file=$<
 
+run-parallels-vm:
+	mkdir $(LIVE).hdd
+	qemu-img convert -O parallels $(LIVE).qcow2 $(LIVE).hdd/live.0.$(PARALLELS_UUID).hds
+	qemu-img info -f parallels --output json $(LIVE).hdd/live.0.$(PARALLELS_UUID).hds | jq --raw-output '.["virtual-size"]' | xargs ./tools/parallels_disk.sh $(LIVE) $(PARALLELS_UUID)
+	prlctl create eve --distribution ubuntu --no-hdd --dst $(DIST)/
```

Comment:
is this `prlctl create` something that needs to be run only once or do we need to run it every time we're trying to run EVE under parallels?

IOW, it seems that you're managing `disk` under `$(DIST)` but the Parallel's VM description and state is still under `$HOME/Parallels/eve.pvm` -- I'm just wondering if it would make sense to `rm -rf $HOME/Parallels/eve.pvm` or maybe there's even a way to tell parallels to manage that state under `$(DIST)` so it becomes `$(DIST)/live.pvm` or something.

Makes sense?

Reply:
1) Yes, `prlctl create` needs once. We are able to change CPU, MEM, and other settings while VM is off without recreating VM again. 
2) For sure we will move pvm inside the $(DIST) folder.

Reply:
Great! Looking forward to your updated PR